lib/pushmi_pullyu/fedora_config.rb

```diff
@@ -0,0 +1,46 @@
+require 'yaml'
+class PushmiPullyu::FedoraConfig
```

Comment:
Maybe. I personally prefer config.fedora.username to config.fedora_username as a conceptual, single-responsibility type of grouping. I had looked at a superclass for config loading to DRY it up a bit, but it wasn't going in a direction I liked (the subclasses still looked unnaturally complicated).